spec/system/dashboards/user_visits_dashboard_spec.rb

```diff
@@ -27,4 +29,29 @@
       end
     end
   end
+
+  context "when looking at actions panel" do
```

Comment:
I notice that this spec file does not cover the Dashboard adequately. I will shore up the specs in this file in a subsequent PR. 👍🏾